exonum/src/blockchain/schema.rs

```diff
@@ -277,12 +277,12 @@ where
 
     /// Returns configuration for given configuration hash.
     pub fn configuration_by_hash(&self, hash: &Hash) -> Option<StoredConfiguration> {
-        self.configs().get(hash)
+        self.configs().get(&EntryHash(*hash))
```

Comment:
This is quite awkward. Could it make sense to use a method signature involving `Borrow` as in [standard collections](https://doc.rust-lang.org/1.21.0/std/collections/struct.BTreeMap.html#method.get)?